chatcli_gpt/chatcli.py

```diff
@@ -353,6 +353,7 @@ def answer(request_messages, model, plugins, stream=True, tags=None):
     else:
         completion = synchroneous_request(request_messages, model)
 
+    # TODO: handle multiple choices
```

Comment:
Interesting. I hadn't planned on doing this because I'm not sure how I'd include multiple conversation branches in the interface / log - and using --retry seems natural.

I can see some potential interfaces which would be cool.

Reply:
Potentially let's find some use cases and see how this would like.
A naive use case is just to save the extra choices in the log files.

Reply:
There shouldn't be extra choices unless we request them. (And the extra choices are saved.)

Reply:
Ack; I would find some time to give a detailed read of the openai API.